app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,8 @@
         android:theme="@style/AppTheme"
         android:usesCleartextTraffic="true"
         tools:ignore="GoogleAppIndexingWarning">
-        <activity android:name=".activities.FilteredTagsActivity"></activity>
+        <activity android:name=".activities.BaseActivity"></activity>
```

Comment:
BaseActivity should not be declared in the manifest since it would never be used by itself ... also make the class abstract if it's not already